src/controllers/dynakube/dynakube_controller.go

```diff
@@ -157,15 +155,17 @@ func (controller *DynakubeController) reconcileDynaKube(ctx context.Context, dkS
 	}
 	dtc, upd, err := dtcReconciler.Reconcile(ctx, dkState.Instance)
 
-	dkState.Update(upd, defaultUpdateInterval, "Token conditions updated")
+	dkState.Update(upd, "Token conditions updated")
 	if dkState.Error(err) {
 		log.Error(err, "failed to check tokens")
 		return
 	}
 	dkState.ValidTokens = true
 	if !dtcReconciler.ValidTokens {
 		dkState.ValidTokens = false
-		log.Info("paas or api token not valid", "name", dkState.Instance.GetName())
+		errMsg := "paas or api token not valid"
+		log.Info(errMsg, "name", dkState.Instance.GetName())
+		dkState.Err = fmt.Errorf(errMsg)
```

Comment:
this shouldn't be set explicitly, but use `dkstate.Error(..)`. Is there a reason that `Err` is even exported?

Reply:
In line 121 we check this error in the dynakube controller so it has to be exported, but I agree, in this case the error function should be called